pegasus/forms/hoc_signup_2014.rb

```diff
@@ -28,16 +28,4 @@ def self.process(data)
       results.merge! location.to_solr if location
     end
   end
-
-  def self.solr_query(params)
```

Comment:
Agree to removing solr_query, but what about these queries?

https://github.com/code-dot-org/code-dot-org/blob/1c6689b3bf3f08c873b9567caef159c53eb58a78/bin/map-hoc-afterschool#L23

https://github.com/code-dot-org/code-dot-org/blob/51c178b6ca7f55c2506d8df79af5adf89db96339/pegasus/sites.v3/hourofcode.com/public/events/2014/splat.haml#L23